client/api_object.go

```diff
@@ -1339,7 +1339,7 @@ func (c *Client) ListObjectPolicies(ctx context.Context, objectName, bucketName
 
 	policies := types.ListObjectPoliciesResponse{}
 	bufStr := buf.String()
-	err = xml.Unmarshal([]byte(bufStr), &policies.Policies)
```

Comment:
Please also add more details in the function doc for actionType parameters

Reply:
added

Reply:
Please also add an e2e test on this API.